salt/states/pkgrepo.py

```diff
@@ -387,6 +402,14 @@ def absent(name, **kwargs):
         if not name.startswith('ppa:'):
             name = 'ppa:' + name
 
+    remove_key = any(kwargs.get(x) is not None
+                     for x in ('keyid', 'keyid_ppa'))
```

Comment:
On 04/21/2015 10:54 PM, Erik Johnson wrote:
> Do you think I should rename the variable? I can see how it could be
> confusing.

I think you can name it like: execute_del_repo_key. But it's not too 
important. :P